DigitalLearningSolutions.Web/Controllers/TrackingSystem/Centre/Administrator/AdministratorController.cs

```diff
@@ -35,6 +39,12 @@ ICommonService commonService
             int page = 1
         )
         {
+            // Query parameter should take priority over cookie value
+            if (!Request.Query.ContainsKey(nameof(filterBy)))

```

Comment:
It would be good to have a comment about why this rather than `if (filterBy != null)`, to stop people from thinking they know better and changing it.

Reply:
Just to completely avoid ambiguity, alright if we change `that is used` -> `filterBy is set to null`?